src/wallet/gtest/test_wallet.cpp

```diff
@@ -2046,21 +2049,21 @@ TEST(WalletTests, UpdatedSaplingNoteData) {
     auto m = GetTestMasterSaplingSpendingKey();
 
     // Generate dummy Sapling address
-    auto sk = m.Derive(0);
+    auto sk = m.Derive(0 | HARDENED_KEY_LIMIT);
```

Comment:
This constant only happens to have the right value. (Same comment on other instances.)